libraries/chain/webassembly/privileged.cpp

```diff
@@ -17,14 +17,14 @@ namespace eosio { namespace chain { namespace webassembly {
    }
 
    void interface::preactivate_feature( legacy_ptr<const digest_type> feature_digest ) {
```

Comment:
Read-only functions must not be allowed to call the `preactivate_feature` host function, along with all of the following ones:

- `set_resource_limits`
- `set_wasm_parameters_packed`
- `set_proposed_producers`
- `set_proposed_producers_ex`
- `set_blockchain_parameters_packed`
- `set_parameters_packed`
- `set_privileged`